cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CommitCoordinator.java

```diff
@@ -114,72 +124,101 @@ public void onFailure(Throwable throwable)
                             Runnable::run);
     }
 
-    private Stream<ListenableFuture<CommitResult>> commit(final Map<RingInstance, ListeningExecutorService> executors,
-                                                  RingInstance instance,
-                                                  Map<String, Range<BigInteger>> uploadRanges) {
-        if (cluster.instanceIsAvailable(instance)) {
-            ListeningExecutorService executorService = executors.computeIfAbsent(instance, inst -> MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(job.getCommitThreadsPerInstance(), ThreadUtil.threadFactory("commit-sstable-" + inst.getNodeName()))));
-            final List<String> allUuids = new ArrayList<>(uploadRanges.keySet());
-            LOGGER.info("Committing UUIDs={}, Ranges={}, instance={}", allUuids, uploadRanges.values(), instance.getNodeName());
+    private Stream<ListenableFuture<CommitResult>> commit(Map<RingInstance, ListeningExecutorService> executors,
+                                                          RingInstance instance,
+                                                          Map<String, Range<BigInteger>> uploadRanges)
+    {
+        if (cluster.instanceIsAvailable(instance))
+        {
+            ListeningExecutorService executorService = executors.computeIfAbsent(instance,
+                    key -> MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(
+                            job.getCommitThreadsPerInstance(),
+                            ThreadUtil.threadFactory("commit-sstable-" + key.getNodeName()))));
+            List<String> allUuids = new ArrayList<>(uploadRanges.keySet());
+            LOGGER.info("Committing UUIDs={}, Ranges={}, instance={}",
+                        allUuids, uploadRanges.values(), instance.getNodeName());
             List<List<String>> batches = Lists.partition(allUuids, job.getCommitBatchSize());
             return batches.stream().map(uuids -> {
-                final String migrationId = UUID.randomUUID().toString();
+                String migrationId = UUID.randomUUID().toString();
                 return executorService.submit(() -> {
-                    final CommitResult commitResult = new CommitResult(migrationId, instance, uploadRanges);
-                    try {
+                    CommitResult commitResult = new CommitResult(migrationId, instance, uploadRanges);
+                    try
+                    {
                         // TODO
```

Comment:
unrelated, but this TODO can be safely removed. I left it here by accident
```suggestion
                    {
```

Reply:
Removed.